.circleci/config.yml

```diff
@@ -1,80 +0,0 @@
-version: 2.1
```

Comment:
I have deleted the Circle CI file in this PR. This causes the build to fail with

> No `.circleci/config.yml` was found in your project. Add a configuration file to your repository or refer to documentation.

but I don't have permission to disable Circle CI on this repo.

![Screenshot at 2023-01-08 19-24-18](https://user-images.githubusercontent.com/6025893/211216319-176abaa1-27cd-4bb1-8b3b-d71330275a4e.png)


Who else has permission to do this?

Reply:
stopped it